packages/phrases/src/locales/de/errors/connector.ts

```diff
@@ -39,6 +39,8 @@ const connector = {
     'Sie können keine mehreren sozialen Connectors haben, die das gleiche Ziel und die gleiche Plattform haben.',
   cannot_overwrite_metadata_for_non_standard_connector:
     "Die 'Metadaten' dieses Connectors können nicht überschrieben werden.",
+  /** UNTRANSLATED */
+  duplicate_connector_name: 'Connector name already exists. Please choose a different name.',
```

Comment:
why not put this in another file as this file is for connectors

Reply:
We've already got a `single-sign-on.ts` which contains the error code of duplicated/forbidden domains there, can put `duplicate_connector_name` error to the file as well.

Reply:
updated